setup.py

```diff
@@ -114,12 +110,11 @@ def __str__(self):
 
 if is_pypy or is_jython:
```

Comment:
Maybe please add an or is_pure check with the os.getenv('PURE_PYTHON') logic copied from zope.interface etc.? So people can ask to skip extension modules.

Reply:
Ah, right, I copy-pasted a few of these before realizing my mistake.  It's a bad idea to check PURE_PYTHON during build time because that will poison pip's wheel caches.